api/admin/controller/__init__.py

```diff
@@ -654,6 +663,184 @@ def error_response(self, problem_detail):
         return Response(html, problem_detail.status_code)
 
 
+class ResetPasswordController(AdminController):
+    FORGOT_PASSWORD_TEMPLATE = SignInController.SIGN_IN_TEMPLATE
+    RESET_PASSWORD_TEMPLATE = SignInController.SIGN_IN_TEMPLATE
+
+    HEAD_TEMPLATE = SignInController.HEAD_TEMPLATE
+
+    RESPONSE_TEMPLATE_WITH_MESSAGE = (
```

Comment:
Yeah, I was thinking about this, even wanted to ask at one of the last syncs/retros, this formatting with `%` and `.format()` in the same code is a bit weird (but does the job). 

In general, I'm not sure what would be the best approach in situations like these where jinja templates would be (I think) better solution but the rest of the code uses other way of doing things so it would make it a bit awkward having two ways of doing the same thing. Should it be best to just rewrite everything to use better approach (jinja here)? But that is a bit out of the scope of this PR.. Not sure 🤷‍♂️

Reply:
💯 Agree that jinja templates are a better way to go forward here. Its a bit out of the scope of this, and the code is already written, but @pcustic @RishiDiwanTT can one of you make a new ticket to replace the cm template system with jinja?

Reply:
Created [notion](https://www.notion.so/lyrasis/Replace-the-CM-HTML-string-formatting-with-Jinja2-templating-engine-a2c20334d497446b8c1ed7855e3268e1)